### PR TITLE
EZP-28739: Add --continue-on-error option & display corrupted content IDs in the search indexer

### PR DESCRIPTION
**Pull request by kmadejski**
…t IDs in the search indexer

JIRA issue: https://jira.ez.no/browse/EZP-28739

This PR contains changed required for changes introduced in: https://github.com/ezsystems/ezpublish-kernel/pull/2221

### CODE REVIEW

File: lib/Indexer.php [hunk 1]
```diff
@@ -22,13 +23,19 @@ class Indexer extends IncrementalIndexer
...
+    protected $logger;
+
     public function __construct(
         LoggerInterface $logger,
         PersistenceHandler $persistenceHandler,
         DatabaseHandler $databaseHandler,
         SolrSearchHandler $searchHandler
     ) {
         parent::__construct($logger, $persistenceHandler, $databaseHandler, $searchHandler);
+        $this->logger = $logger;
```
Comment (andrerom) [inline lib/Indexer.php]: Why is this needed? Seems to be `protected` in Indexer abstract, so we should be able to read it here right?

Comment (kmadejski) [reply lib/Indexer.php]: 🤦‍♂️  - fixed in https://github.com/ezsystems/ezplatform-solr-search-engine/pull/122/commits/057fa745751c88360faa3ff8fd600844a41c876d

**Review by adamwojs — APPROVED** [review]

**Review by konradoboza — APPROVED** [review]
Option's name is `continue-on-error` so the titles of PRs and ticket are a bit confusing. :)
